Components/home.css

```diff
@@ -0,0 +1,298 @@
+body {
+    height: 312vw;
```

Comment:
Never set height like this because vw is different for different screen sizes. In my screen it goes on scrolling even after the footer. You could probably keep min-height as 100vh instead.
![Screenshot (1)](https://user-images.githubusercontent.com/31363773/154133181-d06cab1b-cabe-4ccf-be72-d05348cc4ae6.png)